src/source/mainwindow.cpp

```diff
@@ -247,3 +247,32 @@ void MainWindow::on_actionLogout_triggered()
     emit adminFeaturesToggled(false);
 }
 
+/*!
+ * \brief MainWindow::on_actionAdd_new_stadium_triggered
+ * @brief Adds stdaium with its corresponding attritbutes
+ * @brief Adds edges with weights between Lags Vegas and it surrounding cities.
+ * @brief Adds souviers to Lag Vegas stdium
+ *
+ */
+void MainWindow::on_actionAdd_new_stadium_triggered()
+{
+    db->AddStadium("Las Vegas Stadium","Las Vegas Gamblers","123 Las Vegas Blv,NV 89101","(702) 962-4000","2016-04-11","50,000","Grass",0,"American","Modern");
+    int id =  db->GetStadiumID("Las Vegas Stadium");
+    qDebug()<<"ID; "<<id;
+    db->AddDistance(id,7,325);
+    db->AddDistance(id,22,300);
+    db->AddDistance(id,18,250);
+
+
+    db->AddSouvenir("Las Vegas Stadium","Baseball cap",23.99);
+    db->AddSouvenir("Las Vegas Stadium","Baseball bat",45.39);
+    db->AddSouvenir("Las Vegas Stadium","Team pendant",15.39);
+    db->AddSouvenir("Las Vegas Stadium","Autographed baseball",19.99);
+
```

Comment:
you're missing one souvenir... check out the list of souvenirs in the project requirements:

"The initial souvenir list is a follows
a. Baseball cap			$23.99
b. Baseball bat			$45.39
c. Team pennant			$15.99
d. Autographed baseball		$19.99
e. Team jersey			$85.99"